HOW_WE_USE_GITHUB.md

```diff
@@ -1,7 +1,45 @@
-## The Issue Sorting Process at `conda`
+<!-- absolute URLs -->
+[conda-org]: https://github.com/conda
+[sub-team]: https://github.com/conda-incubator/governance#sub-teams
 
-> **Note:**
-> This process is similar to that of "triaging", but we've chosen to use different terminology because "triaging" is a word related to very weighty topics (*i.e.*, injuries and war) and we would like to be sensitive to those connotations. Additionally, we are taking a more "fuzzy" approach to sorting (*e.g.*, severities may not be assigned, etc.).
+[project-planning]: https://github.com/orgs/conda/projects/2/views/11
+[project-sorting]: https://github.com/orgs/conda/projects/2/views/11
+[project-support]: https://github.com/orgs/conda/projects/2/views/12
+[project-backlog]: https://github.com/orgs/conda/projects/2/views/13
+[project-sprint]: https://github.com/orgs/conda/projects/2/views/14
+
+[docs-toc]: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/
+[docs-actions]: https://docs.github.com/en/actions
+[docs-saved-reply]: https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/creating-a-saved-reply
+
+[workflow-sync]: https://github.com/conda/infra/blob/main/.github/workflows/sync.yml
+[labels-global]: https://github.com/conda/infra/blob/main/.github/global.yml
+
+<!-- relative URLs -->
+[workflow-issues]: /.github/workflows/issues.yml
+[workflow-project]: /.github/workflows/project.yml
+[labels-local]: /.github/labels.yml
+
+## How We Use GitHub
+
+> **Note**
+> For easy navigation use [GitHub's table of contents feature][docs-toc].
+
+This document seeks to outline how we as a community use GitHub Issues to track bugs and feature requests while still catering to development practices & project management (*e.g.*, release cycles, feature planning, priority sorting, etc.).
+
+<!-- only include high-level topics or particularly noteworthy sections here -->
+Topics:
+  - [What is Issue Sorting?](#what-is-issue-sorting)
+  - [Types of tickets](#types-of-tickets)
+    - [Normal Ticket/Issue](#normal-ticketissue)
+    - [Epics](#epics)
+    - [Spikes](#spikes)
+
+
+### What is "Issue Sorting"?
+
+> **Note**
+> "Issue sorting" is similar to that of "triaging", but we've chosen to use different terminology because "triaging" is a word related to very weighty topics (*e.g.*, injuries and war) and we would like to be sensitive to those connotations. Additionally, we are taking a more "fuzzy" approach to sorting (*e.g.*, severities may not be assigned, etc.).
```

Comment:
They mean the same thing, but you're right we should use more straightforward language.

Since this PR is primarily restructuring existing content, let's update the content in a new PR.